### DISCUSSION TITLE
Do you have a Altstore or Scarlet Repo?

### DISCUSSION THREAD

**Discussion by JACKKY04** [arichornlover/YouTubeRebornPlus#51] [Q&A]
.

**Comment by arichornlover** [comment]
yes! but this user is the one maintaining the Altstore repo and it’s honestly cool.

Altstore: https://raw.githubusercontent.com/CercubeSource/CercubePlusAltstore/main/apps.json

and here’s this version for Scarlet below.

Scarlet: https://raw.githubusercontent.com/arichorn/CercubePlusScarlet/main/apps.json

**Reply by JACKKY04** [reply]
Do you have a repo for YTReborn? Not Cercube?

**Reply by arichornlover** [reply]
Oh, sorry! I thought it was gonna be Cercube 😅
https://arichorn.github.io/index.html?source=https://raw.githubusercontent.com/arichorn/arichorn.github.io/main/apps.json

**Reply by JACKKY04** [reply]
Does this source not work for app db? 

Don't worry it's all working now used the direct link. altstore://source?url=https://raw.githubusercontent.com/arichorn/arichorn.github.io/main/apps.json

Ty!

**Reply by arichornlover** [reply]
No problem! Right now I’m editing it so expect issues right now 😅

**Reply by JACKKY04** [reply]
So I’m confused, is this your GitHub or is it shared. I don’t understand why there is cercube 18.20.3 and then YTRebornPlus 18.20.3, and then the repo is completely different?


Edit: It's all a-bit.Confusing you have many different versions of YouTube.

**Reply by arichornlover** [reply]
Yeah I know but there are Altstore repos that aren’t mine. It’s a lot confusing imo but I only have 1 Altstore repo which is this. https://raw.githubusercontent.com/arichorn/arichorn.github.io/main/apps.json
Which just includes the popular tweaks such as YTReborn and other useful apps.
